internal/logger/global_logfile.go

```diff
@@ -0,0 +1,37 @@
+package logger
+
+import (
+	"bufio"
+
+	"github.com/superfly/flyctl/internal/logger/logfile"
+)
+
+var globalLogFile = logFile{
+	destroyed: true,
```

Comment:
does it need a mutex ?

Reply:
I don't think go mutexes require initialization, right?

Reply:
sorry, I blindly asked.

The question was towards protecting the global variable from changing when InitLogFile or CloseLogFile are called.

Reply:
ahhh ok :)
I think it _should_ be fine?